glutton/udp_broker.go

```diff
@@ -0,0 +1,74 @@
+package main
```

Comment:
This can also go into the glutton package. You basically want almost nothing in main.